File: 0268-missing-number/0268-missing-number.cpp
class Solution {
public:
    int missingNumber(vector<int>& zeta) {
        sort(zeta.begin(),zeta.end());
        int boy;
        for(boy=0;boy<zeta.size();boy++){
            if(boy != zeta[boy]){
                return boy;
            }
        }
        return zeta.size();
    }
};

File: 0268-missing-number/0268-missing-number.js
/**
 * @param {number[]} nums
 * @return {number}
 */
var missingNumber = function(nums) {
    let k = nums.sort((a,b)=>a-b)
    let missingNum;
    for(let i=0;i<nums.length;i++){
      if(nums[i]!=i){
        missingNum = i;
        return missingNum
      }
    }
    return nums.length;
};